caktus_primer/static/js/index.js

```diff
@@ -62,33 +62,6 @@ export class Dropdown extends React.Component {
         }
     }
 
-    key(ev) {
```

Comment:
You have removed this method, but `onKeyDown={(ev)=>this.key(ev)}` below still refers to it.